assets/models/system/compvis-stable-diffusion-v1-4/description.md

```diff
@@ -54,6 +147,6 @@ Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-safe-text-to-image" targe
 > - "image1" and "image2" strings are base64 format.
 > - If "nsfw_content_detected" is True then generated image will be totally black.
 
-#### Model inference: visualization for the prompt - "a photograph of an astronaut riding a horse"
+## Model inference: visualization for the prompt - "a photograph of an astronaut riding a horse"
```

Comment:
```suggestion
### Model inference: visualization for the prompt - "a photograph of an astronaut riding a horse"
```

Reply:
can you please get this checked by @tinaem ?

Reply:
sure, message her.